api/resources/component.go

```diff
@@ -55,19 +58,49 @@ type Component struct {
 	Name                   string                 `json:"name"`
 }
 
+type CPUQuantity struct {
+	resource.Quantity
+}
+
+func (c *CPUQuantity) MarshalJSON() ([]byte, error) {
+	capInStr := strconv.FormatInt(c.MilliValue(), 10)
+	return []byte(fmt.Sprintf(`"%sm"`, capInStr)), nil
```

Comment:
discussed with @wlchn and we think it's better to suffix cpuRequest with 'm'.

otherwise, if user input: 1 for  `cpuRequest`, in response it will be: `"cpuRequest": "1000"`, while for memory, if user input: 1 for `memoryRequest`, in response it will still be `"memoryRequest": "1"`, 

the difference will very likely to cause confusion. 

by suffix with 'm' for cpuRequest, user can easily tell why 1 becomes 1000m.

@davidqhr plz check